packages/yew/src/platform/io.rs

```diff
@@ -1,103 +0,0 @@
-//! This module contains types for I/O functionality.
```

Comment:
It seems that this file was moved, rather than deleted. `git mv` would be better in that case since that preserves the git history

Reply:
`fmt` is a complete rewrite of the buffer and not connected to the previous implementation.

I tried to interactively rebase the changes and is having conflicts with every commits after the commit that removed io.rs.